mesonbuild/interpreter.py

```diff
@@ -2567,21 +2567,25 @@ def get_variables(self):
         return self.variables
 
     def check_stdlibs(self):
-        for for_machine in MachineChoice:
+        machine_choices = [MachineChoice.HOST]
+        if self.coredata.is_cross_build():
+            machine_choices.append(MachineChoice.BUILD)
+        for for_machine in machine_choices:
```

Comment:
not something you need to fix, but we should probably have a `coredata.for_machines()` method that iterates all the valid machines in a build, I seem to remember a lot of code like this around.

Reply:
Yeah, thought the same and was surprised to not find it.